web/cobrands/sass/_layout.scss

```diff
@@ -498,6 +498,18 @@ body.twothirdswidthpage {
   }
 }
 
+// Centre the login and password change pages,
+// but keep them narrow to match the report sidebar
+body.authpage {
+  .content {
+    margin-left: auto;
```

Comment:
For consistently, even when left/right have been the same in the stylesheet, as here, I have used `#{$left}` and `#{$right}` (which makes it much easier to search for any cases I missed :) ).